src/unistd.rs

```diff
@@ -557,27 +557,89 @@ pub fn getcwd() -> Result<PathBuf> {
     }
 }
 
+/// Computes the raw UID and GID values to pass to a `*chown` call.
+///
+/// According to the POSIX specification, -1 is used to indicate that owner and group
```

Comment:
This part doesn't belong in the doc comment.  You can make it a regular comment instead.